pkg/proxy/iptables/proxier.go

```diff
@@ -1378,21 +1378,21 @@ func (proxier *Proxier) syncProxyRules() {
 		// Now write loadbalancing & DNAT rules.
 
 		// Firstly categorize each endpoint into three buckets:
-		//   1. all endpoints that are ready and NOT terminating
-		//   2. all endpoints that are local, ready and NOT terminating
-		//   3. all endpoints that are local, ready and terminating.
+		//   1. all endpoints that are ready and NOT terminating.
+		//   2. all endpoints that are local, ready (NOT terminating).
+		//   3. all endpoints that are local, serving and terminating.
```

Comment:
(Another case of a later commit fixing problems with an earlier commit. Please resquash the fixes correctly.)

Aside: related to earlier comment about changing the flags that `endpointInfo` exports; "ReadyTerminating" is a much better name than "ServingTerminating", and yes, it's confusing that you call it "ReadyTerminating" but it means "Serving && Terminating" not "Ready && Terminating", but I feel like the fix for that is to change the `endpointInfo` flag names, not to change the names here...